LoopKitUI/Views/Settings Editors/InsulinModelSelection.swift

```diff
@@ -99,6 +115,22 @@ public struct InsulinModelSelection: View, HorizontalSizeClassOverride {
         }
     }
     
+    private var contentWithSaveButton: some View {
+        VStack {
+            content
+            Button(action: { self.onSave?(self.viewModel.insulinModelSettings) }) {
+                Text(mode.buttonText)
+                    .actionButtonStyle(.primary)
+                    .padding()
+            }
+            .disabled(viewModel.insulinModelSettings == initialInsulinModelSettings)
+            // Styling to mimic the floating button of a ConfigurationPage
```

Comment:
@novalegra ended up doing this too for the insulin model review, might be worth coordinating if her impl can disappear now that this is here